### PR TITLE
FIX: Export configuration and AEDT crash

### PR DESCRIPTION
**Pull request by maxcapodi78**
- export_configuration method was failing under certain circumstances
- aedt_objects were still referenced after a project was closed and could cause crash in AEDT during debug mode.

### CODE REVIEW

File: pyaedt/generic/configurations.py [hunk 1]
```diff
@@ -1700,22 +1700,27 @@ def _export_mesh_operations(self, dict_out):
...
             for mesh in self._app.mesh.meshregions:
-                if mesh.name == "Settings":
+                if mesh.name in ["Settings", "Global"]:
                     args = ["NAME:Settings"]
                 else:
                     args = ["NAME:" + mesh.name, "Enable:=", mesh.Enable]
                 args += mesh.settings.parse_settings_as_args()
-                args += getattr(mesh, "_parse_assignment_value")()
+                if mesh.name not in ["Settings", "Global"]:
+                    args += getattr(mesh, "_parse_assignment_value")()
```
Comment (SMoraisAnsys) [inline pyaedt/generic/configurations.py:1709]: Can you move that line in the previous `else` statement ? You would be in the case were `mesh.name not in ["Settings", "Global"]`. Unless you need this to be added after `mesh.settings.parse_settings_as_args()`

File: pyaedt/generic/configurations.py [hunk 2]
```diff
@@ -1700,22 +1700,27 @@ def _export_mesh_operations(self, dict_out):
...
                 args += ["UserSpecifiedSettings:=", not mesh.manual_settings]
                 mop = OrderedDict({})
                 _arg2dict(args, mop)
-                if self._app.modeler[args[-3][0]].history().command == "CreateSubRegion":
+                if (
+                    mesh.name not in ["Settings", "Global"]
+                    and self._app.modeler[args[-3][0]].history().command == "CreateSubRegion"
+                ):
```
Comment (SMoraisAnsys) [inline pyaedt/generic/configurations.py:1716]: You might also consider moving that above

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, I left minor comments

File: pyaedt/generic/configurations.py (hunk 2 above)
Comment (maxcapodi78) [reply pyaedt/generic/configurations.py:1716]: same comment as above

**Review by Alberto-DM — DISMISSED** [review]
LGTM

**Review by SMoraisAnsys — DISMISSED** [review]

**Review by SMoraisAnsys — APPROVED** [review]

**Pull request merged**